src/main/java/redis/clients/jedis/params/StrAlgoLCSParams.java

```diff
@@ -0,0 +1,82 @@
+package redis.clients.jedis.params;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+import redis.clients.jedis.Protocol;
+import redis.clients.jedis.util.SafeEncoder;
+
+public class StrAlgoLCSParams extends Params {
+
+    private static final String IDX = "idx";
+    private static final String LEN = "len";
+    private static final String WITHMATCHLEN = "withmatchlen";
+    private static final String MINMATCHLEN = "minmatchlen";
+
+    public StrAlgoLCSParams() {
+    }
+
+    public static StrAlgoLCSParams StrAlgoLCSParams() {
+        return new StrAlgoLCSParams();
+    }
+
+    /**
+     * When IDX is given the command returns an array with the LCS length
+     * and all the ranges in both the strings, start and end offset for
+     * each string, where there are matches.
+     * @return StrAlgoParams
+     */
+    public StrAlgoLCSParams idx() {
+        addParam(IDX);
+        return this;
+    }
+
+    /**
+     * When LEN is given the command returns the length of the longest common substring.
+     * @return StrAlgoParams
+     */
+    public StrAlgoLCSParams len() {
+        addParam(LEN);
+        return this;
+    }
+
+    /**
+     * When WITHMATCHLEN is given each array representing a match will also have the length of the match.
+     * @return StrAlgoParams
+     */
+    public StrAlgoLCSParams withMatchLen() {
+        addParam(WITHMATCHLEN);
+        return this;
+    }
+
+    /**
+     * Specify the minimum match length.
+     * @return StrAlgoParams
+     */
+    public StrAlgoLCSParams minMatchLen(long minMatchLen) {
+        addParam(MINMATCHLEN, minMatchLen);
+        return this;
+    }
+
+    public byte[][] getByteParams(byte[]... args) {
```

Comment:
2. Instead of args: `keyword, a, b`.

Reply:
@sazzad16 It doesn't bother me. Your suggestions is good for code style and strict parameters. I agree and have modified the code.